sehee/models/userDao.js

```diff
@@ -0,0 +1,40 @@
+const { appDataSource } = require("./appDataSource");
+
+// 유저회원가입
+const signUp = async (name, email, password, phoneNumber, profileImage) => {
+  try {
+    return await appDataSource.query(
+      `INSERT INTO users(
+        name,
+        email,
+        password,
+        phoneNumber,
+        profileImage
+    ) VALUES (?, ?, ?, ?, ?);
+  `,
+      [name, email, password, phoneNumber, profileImage]
+    );
+  } catch (err) {
+    const error = new Error("INVALID_DATA_INPUT");
+    error.statusCode = 500;
+    throw error;
+  }
+};
+
+const checkIfUserExistsOrNot = async (userId) => {
+  const user = await appDataSource.query(
+    `SELECT EXISTS(
+      SELECT * FROM
```

Comment:
유저에 대한 정보를 모두 가져오지 말고 필요한 컬럼만 가져올 수 있도록 수정해주세요.
```suggestion
      SELECT
         id,
         name,
         email,
         ...
      FROM
```